src/main/java/com/minecolonies/coremod/entity/ai/citizen/lumberjack/Tree.java

```diff
@@ -157,6 +167,8 @@ else if (Compatibility.isSlimeBlock(block))
             checkTree(world, topLog);
             stumpLocations = new ArrayList<>();
             woodBlocks.clear();
+            final Block BottomBlock = world.getBlockState(location).getBlock();
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Rename this local variable name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS00117)